cd/mxnet_lib/dynamic/Jenkins_pipeline.groovy

```diff
@@ -24,7 +24,7 @@
 
 // NOTE: the following variables are referenced in the mxnet_lib_pipeline jenkins file imported bellow
 // libmxnet location
-libmxnet = 'lib/libmxnet.so'
+libmxnet = 'build/libmxnet.so'
```

Comment:
This is for switching to the cmake build on CD. Should it be done separately / only for 1.x?

Reply:
@mseth10

Reply:
Reverted this change.